src/components/SideBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Sidebar = () => {
+  return (
+    <div className="absolute bg-dark-gray w-64 h-screen left-7 shadow-md pr-7">
+      <ul className="list-none p-0 mt-28">
+        <li>
+          <p className="text-light-gray font-light">PAGES</p>
+        </li>
+        <li className="py-2 mt-4 pt-6 bg-darker-gray">
+          <Link
+            to="/home"
+            className="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white mr-4"
+          >
+            Home
+          </Link>
+        </li>
+        <li className="py-2 mt-4 pt-6 bg-darker-gray">
+          <Link
+            to="/createInvestor"
+            className="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white mr-4"
+          >
+            Create Investor
+          </Link>
+        </li>
+        <li className="py-2 mt-4 pt-6 bg-darker-gray">
+          <Link
+            to="/backtesting"
+            className="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white mr-4"
+          >
+            Back Testing
+          </Link>
+        </li>
+        <li className="py-2 mt-4 pt-6 bg-darker-gray">
+          <Link
+            to="/history"
+            className="block mt-4 lg:inline-block lg:mt-0 text-white hover:text-white mr-4"
+          >
+            Transaction History
+          </Link>
+        </li>
+        <li className="py-2 mt-4 pt-6 bg-darker-gray">
+          <Link
```

Comment:
Do we need a 'My Account' on the NavBar and the sidebar? Seem like just having it on the navbar wouldb e sufficient.

Reply:
This is how it is set up in the Figma prototypes. Do we want to change it to be only sign out now? I was thinking the My Profile will be a dropdown menu where the user can have the option to sign out, and will still have the option to access that page. We did discuss earlier in the semester that it's okay to have multiple ways a user can go to a page.

Reply:
Thats fine because its out of the scope of this story , will worry about that later.